Header Header
int16 Group_number
geometry_msgs/Point Object_position
string Bag_name
int8 Thermal_label
